sdk/couchbase/src/subdoc/lookup_in_specs.rs

```diff
@@ -0,0 +1,97 @@
+use crate::subdoc::lookup_in_specs::LookupInOpType::{Count, Exists, Get};
+use couchbase_core::memdx::subdoc::{LookupInOp, SubdocOp, SubdocOpFlag};
+use typed_builder::TypedBuilder;
+
+pub struct LookupInSpec {
```

Comment:
Everything in here probably needs to be non_exhaustive.

Reply:
Done